carbon-design-system/carbon-for-ibm-dotcom
# fix(masthead): hide scroller buttons when there are no menuItems

### Related Ticket(s)

[ADMCS-6215](https://jsw.ibm.com/browse/ADCMS-6215)

### Description

This bug fix covers the edge case of the `l0Data` on the `<c4d-masthead-l1>` component being undefined or empty.

### Testing instructions

* Open up the Masthead > With L1 story
* Check off the "Use empty data for L1 menu items". This will employ an L1 dataset that does not have any `menuItems` defined.
* The scroller buttons should be hidden

### Changelog

**Changed**

- Handle the edge case of the `l0Data` being `undefined`.
